src/mlpack/methods/naive_bayes/nbc_main.cpp

```diff
@@ -112,8 +111,11 @@ PARAM_FLAG("incremental_variance", "The variance of each class will be "
 
 // Test parameters.
 PARAM_MATRIX_IN("test", "A matrix containing the test set.", "T");
+// parameter output is to be deprecated and removed in mlpack 4.0
```

Comment:
I know this is picky, but can we be sure to capitalize and punctuate the sentence here so it matches mlpack commenting guidelines?  Like this:

```
// The parameter 'output' is deprecated and will be removed in mlpack 4.0.0.
```

Thanks! :+1: